javascript-practice/src/js/controllers/router.js

```diff
@@ -0,0 +1,58 @@
+import { productController } from '../main';
```

Comment:
Should create a folder `routes`  to config route, not put it into controller folder